staging/gatekeeper/templates/service.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   - port: {{ .Values.service.port }}
     targetPort: http
     protocol: TCP
-    name: http
+    name: https
```

Comment:
This seems inconsistent -- `targetPort: http ... name: https`. Is this intentional? Should it be the same, both corrected to `https`?

Reply:
I am not sure. Unlike `name` which is just a hint, `targetPort` is the actual port on which service is listening. I don't know enough to change the listening port.

Reply:
https://github.com/mesosphere/charts/blob/813c68dd6f8c8811e56ec243a74e3ae04d451dfc/staging/gatekeeper/templates/deployment.yaml#L48

Reply:
Isn't this actually http though?

Reply:
@joejulian yeah, it is `http`. However, the issue is here https://github.com/mesosphere/charts/blob/master/staging/gatekeeper/values.yaml#L39. Why that needs to be `443`?

Reply:
If it's http, why does it have all the certificate files?
https://github.com/mesosphere/charts/blob/813c68dd6f8c8811e56ec243a74e3ae04d451dfc/staging/gatekeeper/templates/deployment.yaml#L63-L68

I didn't think webhooks were allowed to _not_ be https.

Reply:
As an aside, the new upstream chart doesn't allow the service port to be specified in the values.

Reply:
Thats nice. I guess the right course of action in that case is to just update the chart. I'll make necessary changes in this PR.